netweaver/setup/install_pydbapi.sls

```diff
@@ -0,0 +1,53 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+
+{% for node in netweaver.nodes if node.host == host and node.sap_instance in ['db', 'pas', 'aas'] %}
+{% if loop.first %}
+{% set pydbapi_output_dir = '/tmp/pydbapi' %}
+nw_install_python_pip:
+  pkg.installed:
+    {% if grains['pythonversion'][0] == 2 %}
+    - name: python-pip
+    {% else %}
+    - name: python3-pip
+    {% endif %}
+    - retry:
+        attempts: 3
+        interval: 15
+    - resolve_capabilities: true
+
+nw_extract_pydbapi_client:
+  hana.pydbapi_extracted:
+    - name: PYDBAPI.TGZ
+    - software_folders: {{ netweaver.additional_dvds }}
+    - output_dir: {{ pydbapi_output_dir }}
+    - hana_version: '20'
+    - force: true
+
+# pip.installed fails as it cannot manage propler file names with regular expressions
+# TODO: Improve this to use pip.installed somehow
+nw_install_pydbapi_client:
```

Comment:
To simplify this, maybe you could search the exact file name first and then call the pip state.

Reply:
Is this simpler or more complicated? hehe
Remember that we cannot store variables from salt execution modules. So, I would need to:
- Create a new execution module to do so
- Create some ugly bash script line to find the name (I don't know if this would work)

Reply:
I didn't remember because I didn't know 😆